src/settings/tabrepository.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TABREPOSITORY_H
+#define TABREPOSITORY_H
+
+#include <QWidget>
+
+class TabRepository : public QWidget
+{
+    Q_OBJECT
+public:
+    explicit TabRepository(QWidget *parent = nullptr);
+private:
+    // design parameters
+    int mBasicStretch                = 1;
+
+signals:
```

Comment:
why do you need signals? you don't have them

Reply:
this tab not implemented yet
so after implementation they will